src/main/histFileMod.F90

```diff
@@ -84,75 +82,86 @@ module histFileMod
   character(len=max_namlen), public :: &
        hist_type1d_pertape(max_tapes)  = (/(' ',ni=1,max_tapes)/)   ! namelist: per tape type1d
 
-  character(len=max_namlen+2), public :: &
-       fincl(max_flds,max_tapes)         ! namelist-equivalence list of fields to add
+  logical, public :: &
+       hist_empty_htapes  = .false.      ! namelist: disable default-active history fields (which
+                                         ! only exist on history tape 1). Overridden by hist_fincl1 
+                                         ! flag.
 
   character(len=max_namlen+2), public :: &
-       hist_fincl1(max_flds) = ' '       ! namelist: list of fields to add
+       hist_fincl1(max_flds) = ' '       ! namelist: list of fields to include in history tape 1
```

Comment:
I wonder if you should also include something that says "the "h0" set of files" or something to that effect?

Reply:
Yes, exactly, we call it "history tape 1", but the filenames are actually labeled as "h0". This is kind of an awkward thing we've been preserving. I like what you added below for history_tape and the breadcrumbs you put for pointing to it. Maybe there could be another one on top of the hist_fincl and hist_fexcl definitions?

Reply:
SGTM. How do I add a change at this point - looks like this commit got pulled into PR #2031, so should I just send a new PR for this final thing?

Reply:
You could either: create a new PR, add it to #2022 which is still outstanding, or add it to here, and this PR will stay open and I can pull it into another tag. I have a slight preference for adding to #2022, but any of these would work fine. So I don't care too much.

Because we run a bunch of testing by hand to create our tags, I am very likely to merge a few PR's together to create the tag (especially if they are simple bit-for-bit type things). So you should expect that. But, it's also easier to see things broken up into smallish PR's so you can see what's going on with a specific PR. So I do still like having things broken up into separate PR's.

Reply:
Added PR #2034 (note its commit is on top of the commits in this PR)